S3 endpoint doesn't allow containing the scheme

May I have a question that why fybrik doesn't allow S3 endpoint containing the scheme i.e. "http://" ?
https://github.com/fybrik/fybrik/pull/874/files

In a DEV/test environment, we use localstack to setup a local S3 service, if the endpoint doesn't contain the scheme "http", the data access modules ( arrow flight module or mover) have to request data by "https" , but this will fail due to the default SSL certificate mismatch.

```
Caused by: javax.net.ssl.SSLPeerUnverifiedException: Certificate for <localstack.fybrik-notebook-sample.svc.cluster.local> doesn't match any of the subject alternative names: [*.cloudfront.localhost.localstack.cloud, *.execute-api.localhost.localstack.cloud, *.localhost.localstack.cloud, *.s3.localhost.localstack.cloud, *.scm.localhost.localstack.cloud, localhost.localstack.cloud]
```


Below is my workaround, it works with the localstack environment.
```patch
diff --git a/manager/controllers/app/plotter_generator.go b/manager/controllers/app/plotter_generator.go
index 354f2c29..3e842d50 100644
--- a/manager/controllers/app/plotter_generator.go
+++ b/manager/controllers/app/plotter_generator.go
@@ -69,17 +68,17 @@ func (p *PlotterGenerator) GetCopyDestination(item DataInfo, destinationInterfac

        connection := taxonomy.Connection{
                Name: "s3",
                AdditionalProperties: serde.Properties{
                        Items: map[string]interface{}{
                                "s3": map[string]interface{}{
-                                       "endpoint":   url.Host,
+                                       "endpoint":   bucket.Endpoint,
                                        "bucket":     bucket.Name,
                                        "object_key": genObjectKeyName,
                                },

```

--- a/manager/controllers/app/plotter_generator.go
+++ b/manager/controllers/app/plotter_generator.go
@@ -5,7 +5,6 @@ package app
 
 import (
 	"bytes"
-	"net/url"
 	"strings"
 	"text/template"
 
@@ -67,19 +66,12 @@ func (p *PlotterGenerator) GetCopyDestination(item DataInfo, destinationInterfac
 		return nil, err
 	}
 
-	// S3 endpoint should not include the url scheme only the host name
-	// thus ignoring it if such exists.
-	url, err := url.Parse(bucket.Endpoint)
-	if err != nil {
-		return nil, err
-	}
-
 	connection := taxonomy.Connection{
 		Name: "s3",
 		AdditionalProperties: serde.Properties{
 			Items: map[string]interface{}{
 				"s3": map[string]interface{}{
-					"endpoint":   url.Host,
+					"endpoint":   bucket.Endpoint,
 					"bucket":     bucket.Name,
 					"object_key": genObjectKeyName,
 				},

--- a/pkg/connectors/protos/dataset_details.proto
+++ b/pkg/connectors/protos/dataset_details.proto
@@ -38,7 +38,7 @@ message Db2DataStore {
 }
 
 message S3DataStore {
-    string endpoint = 1;    // endpoint should contain only the host name without the scheme i.e., "s3.eu-gb.cloud-object-storage.appdomain.cloud"
+    string endpoint = 1;
     string bucket = 2;
     string object_key = 3; //can be object name or the prefix for dataset
     string region = 4;     // WKC does not return it, it will stay empty in our case!!!

--- a/site/docs/reference/connectors.md
+++ b/site/docs/reference/connectors.md
@@ -326,7 +326,7 @@
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| endpoint | [string](#string) |  | endpoint should contain only the host name without the scheme i.e., "s3.eu-gb.cloud-object-storage.appdomain.cloud" |
+| endpoint | [string](#string) |  |  |
 | bucket | [string](#string) |  |  |
 | object_key | [string](#string) |  | can be object name or the prefix for dataset |
 | region | [string](#string) |  | WKC does not return it, it will stay empty in our case!!! |

